docs-2.0/7.data-security/2.backup-restore/1.what-is-br.md

```diff
@@ -23,17 +23,30 @@ This section introduces how backup and restoration are implemented in the BR.
 
 To back up data, the BR sends a backup request to each metad process to trigger the backup process as follows:
 
-1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if remote store such as Amazon S3 or Alibaba Cloud OSS is necessary, their client installation and configuration is verified.
+1. The SSH login from the BR to the meta and the storage servers is verified. Besides, if a remote storage system such as Amazon S3 or Alibaba Cloud OSS is necessary, its client installation and configuration is verified.
+
 2. The BR sends a request to create backup files.
-3. The metad process is locked.
-4. The metad process blocks writes to the specified graph spaces.
-5. The metad process sends a request to storaged processes for the snapthot file names.
-6. The metad process scans local RocksDB files and outputs SST files.
-7. The metad process cancels blocking writes to the graph spaces.
-8. The metad process sends response to the BR with the meta data including the thrift format, partition information of the graph spaces, and the log commit ID of each partition, and the snapshot information including the catalog of the snapshots of each storaged process, their SST file names of the meta server, and the backup file names.
-9. The metad process is unlocked.
-10. The BR SSHs to the meta server where the leader locates and all the storage servers, and backs up files. If Amazon S3 or Alibaba Cloud OSS is used, the backup files are uploaded to the remote store.
-11. If Amazon S3 or Alibaba Cloud OSS is used, the BR calls commands to upload files to the remote store.
+
+3. The metad processes are locked.
+
+4. The metad processes block writing to the specified graph spaces.
+
+5. The metad processes send a request to the storaged processes for the snapthot file names.
+
+6. The metad processes scan local RocksDB files and output SST files.
+
+7. The metad processes cancel blocking writing to the graph spaces.
+
+8. The metad processes send responses to the BR with the meta data including:
+   - the thrift format, partition information of the graph spaces, and the log commit ID of each partition
+   - the snapshot information including the catalog of the snapshots of each storaged process, their SST file names of the meta server, and the backup file names.
+
+9. The metad processes are unlocked.
+
+10. The account on the BR machine logs on via SSH to the meta server where the leader locates and all the storage servers, and backs up files.
+
+11. If Amazon S3 or Alibaba Cloud OSS is used, the BR calls commands to upload files to the remote storage system.
+
```

Comment:
> This Question is very important, since it happens a lot.

The backup process must be restarted. For now, users cannot resume the backup process from the broken point.